deployment/test/flyte_generated.yaml

```diff
@@ -3,161 +3,1301 @@ kind: Namespace
 metadata:
   name: flyte
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This diff is large because I consolidated the contour-ingress-controller manifest between `test` and `sandbox`. @kumare3: Do you have context about why these were different to begin with? Should we keep them different?

Reply:
yes, Sandbox is used by users to try out. test on the other hand has no flyteconsole and is used to run end to end tests.

Reply:
@kumare3: Hmm I see, we should probably tweak the test environment then. We can probably get rid of `contour_ingress_controller` there as well, then?

Reply:
ohh that would be great, because that will speed up the end to end. I think for the test environment eventually I prefer using the sandbox docker image itself as that would be much faster to startup and run tests